video/agon.h

```diff
@@ -26,6 +26,8 @@
 #define UART_RTS				13		// The ESP32 RTS pin (eZ80 CTS)
 #define UART_CTS	 			14		// The ESP32 CTS pin (eZ80 RTS)
 
+#define COMMS_TIMEOUT			10		// Timeout for VDP commands (ms)

```

Comment:
this was an experimental value to change the timeout on VDU reads.  10ms is a bit too quick, as it makes timeouts more likely when attempting to stream data across from BBC BASIC, especially if an SD card read happens as part of that.

#30 changes this value to 100

ideally we should improve our use of timeouts, as well as try to remove uses of blocking comms, as noted in #35